week-1/refactor/find.js

```diff
@@ -1,13 +1,19 @@
 // Refactor the implementation of find to use a for...of loop
 
 function find(list, target) {
-  for (let index = 0; index < list.length; index++) {
-    const element = list[index];
-    if (element === target) {
-      return index;
+  for (const e of list) {
+    if (e === target) {
```

Comment:
I would advice you to use more discriptive name for variale inside the  for of loop instead of `e`